<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsDlgPgBufferBase</class>
 <widget class="QDialog" name="QgsDlgPgBufferBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Buffer features</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="txtBufferedLayerName" />
      </item>
      <item row="4" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>93</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="textLabel2_2_2" >
        <property name="text" >
         <string>Geometry column:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3" >
       <widget class="QCheckBox" name="chkAddToMap" >
        <property name="text" >
         <string>Add the buffered layer to the map?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="txtBufferDistance" />
      </item>
      <item row="6" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>158</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel2_2" >
        <property name="text" >
         <string>Spatial reference ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="txtSrid" />
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>93</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>Schema:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Unique field to use as feature id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>Table name for the buffered layer:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>93</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>93</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3" >
       <widget class="QComboBox" name="cmbFields" >
        <item>
         <property name="text" >
          <string>Create unique object id</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLineEdit" name="txtGeometryColumn" />
      </item>
      <item row="5" column="1" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>167</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>93</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="3" >
       <widget class="QComboBox" name="cmbSchema" >
        <item>
         <property name="text" >
          <string>public</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Buffer distance in map units:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblBufferInfo" >
     <property name="text" >
      <string>&lt;h2>Buffer the features in layer: &lt;/h2></string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnHelp" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
       <property name="shortcut" >
        <string>F1</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>QgsDlgPgBufferBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>371</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel" >
     <x>140</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QgsDlgPgBufferBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>480</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel" >
     <x>237</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
